astropy/utils/tests/test_data.py

```diff
@@ -57,7 +57,7 @@
     download_files_in_parallel,
 )
 
-TRAVIS = os.environ.get('TRAVIS', False)
+TRAVIS = os.environ.get('TRAVIS', False) == "true"
```

Comment:
Do we need to do this in other places too?

Reply:
Not really, as the other places pass in a string into `skipif` condition but I am not.